examples/rmq_broadcast_client.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+import sys
+
+from kiwipy import rmq
+
+# pylint: disable=invalid-name
+
+body = ' '.join(sys.argv[1:]) or "___"
+
+with rmq.RmqThreadCommunicator.connect(connection_params={'url': 'amqp://127.0.0.1'}) as communicator:
+    # send message with different sender and subject
+
+    # listen by two subscriber
+    sendr = 'bob.jones'
+    subj = 'purchase.car'
+    communicator.broadcast_send(body, sender=sendr, subject=subj)
+    # sender filtered
+    sendr = 'bob.smith'
+    subj = 'purchase.car'
+    communicator.broadcast_send(body, sender=sendr, subject=subj)
+    # subject filterd
+    sendr = 'bob.jones'
+    subj = 'sell.car'
+    communicator.broadcast_send(body, sender=sendr, subject=subj)
```

Comment:
Do I understand correctly that this example is just 3 messages being sent?
If yes, then I guess two messages would already be enough to illustrate the concept.

Reply:
Yes, but I keep three messages here just because class `BroadcastFilter` have two separated method `add_subject_filter` and `add_sender_filter`.

Maybe these two methods can be merged?

Reply:
I noticed only now the changes in the comment lines in the last two cases.
Just from reading the code, I think it's still not easy to understand what exactly is going on, and why e.g. the last two result in different types of filtering.

Reply:
The difference between the two filters is reflected on the server side. It is true that there's no different from the client side(client only send the message), and as you said, there's really no need to emphasize the difference between the two filters. 
Therefore, I'll remove the sender filter for clarity.